pkg/beacon/majority.go

```diff
@@ -204,6 +215,41 @@ func (m *Majority) checkFinality(ctx context.Context) error {
 	return nil
 }
 
+func (m *Majority) checkBeaconStateSpec(ctx context.Context) error {
+	// No-Op if we already have a beacon state spec
+	if m.spec != nil {
+		return nil
+	}
+
+	m.log.Debug("Fetching beacon state spec")
+
+	readyNodes := m.nodes.Ready(ctx)
```

Comment:
can i omit this block and just rely on the next one?

Reply:
Yup